std_msgs/Header header
nav_msgs/MapMetaData info
int8[] polygonType
int32[] polygonId
topoPolygon[] polygons
